.github/workflows/superlinter.yml

```diff
@@ -13,10 +13,28 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Install, test and lint webapp
-      working-directory: ./news-list
+      working-directory: ./webapp
       run: |
         npm install
         npm run test:unit
         npm run lint
       env:
         CI: true
+
+  build-backend:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install, test and lint backend
+      working-directory: ./backend
+      run: |
+        npm install
+        npm run test
+        npm run lint
+      env:
+        CI: true
```

Comment:
⭐ For correct folder structure, basic apollo server setup and it's linter in the CI/CD pipeline. ✅
⭐ For the apollo-server-testing setup and its integration in the CI/CD pipeline.✅